tf/train.py

```diff
@@ -158,8 +164,9 @@ def main(cmd):
 
     shuffle_size = int(shuffle_size*(1.0-train_ratio))
     if experimental_parser:
-        test_dataset = tf.data.Dataset.from_tensor_slices(test_chunks).shuffle(len(test_chunks)).repeat()\
-                         .interleave(lambda x: tf.data.FixedLengthRecordDataset(x, 8308, compression_type='GZIP', num_parallel_reads=1).filter(sample), num_parallel_calls=tf.data.experimental.AUTOTUNE)\
+        test_dataset = tf.data.Dataset.from_tensor_slices(test_chunks).shuffle(len(test_chunks)).repeat().batch(256)\
```

Comment:
Why escape end of line? Leaving unmatched parentheses is more idiomatic way to do multiline python statements.
Also here `yapf` would also help. :-P